openshift_sti_images/nodejs.adoc

```diff
@@ -0,0 +1,130 @@
+= NodeJS
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+Our OpenShift https://github.com/openshift/sti-nodejs/tree/master/0.10[NodeJS] image repository contains the sources and Dockerfiles for building various versions of NodeJS platform images. The resulting images can be run either by Docker or using https://github.com/openshift/source-to-image[STI].
+
+== Versions
+Current supported versions of NodeJS:
+
+* https://github.com/openshift/sti-nodejs/tree/master/0.10[0.10]
+
+== Base Images
+
+* RHEL-7
+* CentOS-7
+
+[NOTE]
```

Comment:
"Admonition blocks" like this must be formatted in one of two ways:

Option 1 is all in one line: `[NOTE]: Foo bar, baz.`

Option 2 is as a block:

    [NOTE]
    ====
    Foo bar, baz.
    ====

So please update this to match one of these patterns.